db/migrate/20251116000003_create_registration_policies.rb

```diff
@@ -0,0 +1,25 @@
+class CreateRegistrationPolicies < ActiveRecord::Migration[8.0]
+  def change
+    create_table :registration_policies do |t|
+      t.references :registration_campaign,
+                   null: false,
+                   foreign_key: true,
+                   index: true
+      t.integer :kind, null: false
+      t.integer :phase, null: false, default: 0
+      t.integer :position
+      t.boolean :active, null: false, default: true
```

Comment:
We enforce `null: false` for `:active`, but don't have a respective validation in the model.

Reply:
Fixed.